server/src/test/java/io/crate/integrationtests/AggregateExpressionIntegrationTest.java

```diff
@@ -238,4 +238,31 @@ public void test_aggregation_in_order_by_without_having_them_in_the_select_list(
                       "b\n" +
                       "c\n"));
     }
+
+    @Test
+    public void test_aggregations_on_literal() {
```

Comment:
Can this be moved into a `DocValuesAggregatesTest` test case?

Reply:
I fixed an existing test to catch that. Imho I would keep the integ test as well, to have one end-to-end query (ok it's actually 2 :)) that checks aggs on literals, wdyt?

Reply:
As discussed offline, I'll remove the integ test here, and add something in crate-qa (sql-logic if possible?).